src/Orbit/main.cpp

```diff
@@ -157,7 +159,7 @@ static QStringList ExtractCommandLineFlags(const std::vector<std::string>& comma
   QStringList command_line_flags;
   absl::flat_hash_set<std::string> positional_arg_set(positional_args.begin(),
                                                       positional_args.end());
-  for (std::string command_line_arg : command_line_args) {
+  for (const std::string& command_line_arg : command_line_args) {
```

Comment:
Nit: Maybe use `string_view` instread of `const std::string&`.

Reply:
command_line_args is a vector of strings.. what are advantages of using string_view in this case?

Reply:
I was thinking that `const std::string&` can be replace by `string_view` as the latter one would be faster in general cases (go/totw/179). 

But I agree that there is no advantage in this case.